automated-tests/src/main/java/com/hp/ov/sdk/network/FcNetworkResource.java

```diff
@@ -115,4 +122,26 @@ private FcNetwork builderUpdate(FcNetwork fcNetwork) {
         return fcNetwork;
     }
 
+	public String patchFcNetwork(String id) {
+		FcNetwork fcNetwork = new FcNetwork();
+		fcNetwork.setName(resourceProperties.get("name"));
+
+        Patch patch = new Patch();
+
+        patch.setOp(Patch.PatchOperation.valueOf(resourceProperties.get("op")));
+        patch.setPath(resourceProperties.get("path"));
+        List<String> scopeUris = fcNetwork.getScopeUris();
+        scopeUris.add(SCOPES_URI + "/" + getScopeId(resourceProperties.get("value")));
+        patch.setValue(scopeUris);
+
+        return objToString(client.patch(id, patch));
+	}
+
+    private String getScopeId(String scopeName) {
+        ScopeClient scopeClient = oneViewClient.scope();
+
+        Scope scope = scopeClient.getByName(scopeName).get(0);

```

Comment:
Please add a verification if scope exists, to avoid a NPE.

Reply:
Done